src/formatDate.js

```diff
@@ -8,7 +8,75 @@
  * @returns {string}
  */
 function formatDate(date, fromFormat, toFormat) {
-  // write code here
-}
+  // create property for symbols from and to
+  const simbolFormat = fromFormat[fromFormat.length - 1];
+  const simbolTo = toFormat[toFormat.length - 1];
+  // create split array from date
+  const dateDay = date.split(simbolFormat);
+  // create property that will save new date
+  let year = '';
+  let day = '';
+  let month = '';
```

Comment:
Be cautious with the initial empty string values for 'year', 'day', and 'month'. If the 'fromFormat' does not contain one of these components, they will remain empty strings, which may not be the intended behavior when building the 'newDate'.